### DISCUSSION TITLE
I have an error trying to create Uno App

### DISCUSSION THREAD

**Discussion by Kureshik** [unoplatform/uno#12569] [Q&A] [answered]
Hi, I just started with the Uno Platform. Now I have error from VS "The value does not fall within the expected range" when I create new Uno Platform App. What's wrong?

**Comment by jeromelaban** [comment]
Thanks for the report! Could you indicate where you're seeing this, and which version of visual studio you are using (which 17.x) ?

**Reply by Kureshik** [reply]
I see this after I click the create button in the Uno Platform Template Wizard, no matter which preset I choose.
My vs version is 17.6.2

**Reply by jeromelaban** [reply]
Thanks for the update. Could you tell us which language your system is using ? Could you also, using the file explorer, take a screenshot the directory where the project is supposed to be created?

/cc @nickrandolph

**Reply by Kureshik** [reply]
Russian lang,
![image](https://github.com/unoplatform/uno/assets/79392961/52ecf331-f8d7-462c-a6b0-3458c47c8b99)

**Reply by jeromelaban** [reply]
Thanks, which project name did you try to use? Is it in the folder for which you've taken the screen shot?

**Reply by Kureshik** [reply]
Currently no. I have tried "UnoApp1", "Uno Test app" and "UnoApp test". They all appeared in this folder during the error, I then deleted them

**Reply by jeromelaban** [reply]
Thanks. If you try creating with the `UnoApp1` name, can you tell what's created on disk?

**Reply by Kureshik** [reply]
Creates an empty folder named UnoApp1

**Comment by tenderodigital** [comment]
Hello: I have the same error. I work in Spanish. Just in case I have completely uninstalled Uno Platform and Visual Studio before. I have reinstalled everything again and the error persists. When creating a new Uno Platform application after selecting the options, a dialog box appears: "The value is not within the expected range"
I have left the default folder name and application name options. The app is called unoapp2.
And it also creates for me an empty folder on the disk with the name of the application.
The error box is this:
![image](https://github.com/unoplatform/uno/assets/135991332/425dcf85-f558-40dd-a0c2-fd540e738fc8)

![image](https://github.com/unoplatform/uno/assets/135991332/95319883-4053-4a47-86ef-53f32a4b1ce9)

For what it's worth, I have the error since the wizard has more options, specifically 10:

![image](https://github.com/unoplatform/uno/assets/135991332/12ffa292-2deb-4659-ac43-c0253e48a9a7)

**Reply by jeromelaban** [reply]
Thanks for the details. I've opened https://github.com/unoplatform/uno/issues/12578, let's continue the conversation there.

**Comment by tenderodigital** [comment]
I had forgotten before. I have been using Uno Platform for 3 months without any problems until today.

**Reply by joseluisct** [reply]
Same problem here

**Reply by jeromelaban** [reply, marked as answer]
A fix is on the way, it should be published in the next few hours. Thanks everyone for the help!
